externs/closure-compiler.js

```diff
@@ -15,3 +15,13 @@
  * @see http://www.w3.org/TR/pointerevents/#the-touch-action-css-property
  */
 CSSProperties.prototype.touchAction;
+
+/**
+ * @type {number}
+ */
+WheelEvent.DOM_DELTA_LINE;
```

Comment:
Awesome @fredj, thanks! I was surprised to see that the compiler knows about it.